app/rental_routes.py

```diff
@@ -0,0 +1,81 @@
+from app import db
+from app.models.customer import Customer
+from app.models.video import Video
+from app.models.rental import Rental
+from app.validation_fns import validate_model, validate_request_data_and_create_obj
+from flask import Blueprint, jsonify, abort, make_response, request
+
+rentals_bp = Blueprint("rentals", __name__, url_prefix="/rentals")
+
+
+
+##################
+##  RENTAL ROUTES  ##
+##################
+
+@rentals_bp.route("/check-out", methods=["POST"])
+def create_one_rental():
+    request_body = request.get_json()
+
+    # rental request MUST have customer_id and video_id included
+    try:
+        video_id = request_body["video_id"]
+        customer_id = request_body["customer_id"]
+    except KeyError as e:
+        key = str(e).strip("\'")
+        abort(make_response(jsonify({"details": f"Request body must include {key}."}), 400))
+
+    rental_customer = validate_model(Customer, customer_id)
+    rental_video = validate_model(Video, video_id)
+    videos_checked_out_count = rental_customer.rental_count() + 1
+    available_inventory = rental_video.available_inventory() - 1
```

Comment:
Okay, so this makes sense as to why video has a relationship back to rentals.  Nice job using the OOP-style interface provided through SQLAlchemy!